README.md

```diff
@@ -27,7 +27,7 @@ also capable of comparing the level of performance recorded in two different git
 This can be used for example to validate a feature branch against the main branch, perhaps
 integrated with a pull request.
 
-See the documentation in [docs/README.md](docs/README.md).
+See the documentation in https://otava.apache.org/docs/overview/.
```

Comment:
This change is required because the relative url leads to 404 from the PyPI page. I figured that it's best to lead to the rendered docs than to Github.